README.md

```diff
@@ -217,8 +224,7 @@ config = SedonaContext.builder().config("fs.s3a.aws.credentials.provider", "org.
 sedona = SedonaContext.create(config)
 
 df = sedona.read.format("parquet").load("s3a://overturemaps-us-west-2/release/2023-10-19-alpha.0/theme=places/type=place")
```

Comment:
```suggestion
df = sedona.read.format("parquet").load("s3a://overturemaps-us-west-2/release/<release-version>/theme=places/type=place")
```

Reply:
Let us continue this conversation post M2 release since we need to complete this PR for the M2 release today.